src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -0,0 +1,55 @@
+// <copyright file="ConsoleLogRecordExporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET461 || NETSTANDARD2_0
+using System;
+using OpenTelemetry.Logs;
+
+namespace OpenTelemetry.Exporter.Console
+{
+    public class ConsoleLogRecordExporter : ConsoleExporter<LogRecord>
+    {
+        public ConsoleLogRecordExporter(ConsoleExporterOptions options)
+            : base(options)
+        {
+        }
+
+        public override ExportResult Export(in Batch<LogRecord> batch)
+        {
+            var rightPaddingLength = 30;
+            foreach (var logRecord in batch)
+            {
+                this.WriteLine($"{"LogRecord.TraceId:".PadRight(rightPaddingLength)}{logRecord.TraceId}");
+                this.WriteLine($"{"LogRecord.SpanId:".PadRight(rightPaddingLength)}{logRecord.SpanId}");
+                this.WriteLine($"{"LogRecord.Timestamp:".PadRight(rightPaddingLength)}{logRecord.Timestamp:yyyy-MM-ddTHH:mm:ss.fffffffZ}");
+                this.WriteLine($"{"LogRecord.EventId:".PadRight(rightPaddingLength)}{logRecord.EventId}");
+                this.WriteLine($"{"LogRecord.CategoryName:".PadRight(rightPaddingLength)}{logRecord.CategoryName}");
+                this.WriteLine($"{"LogRecord.LogLevel:".PadRight(rightPaddingLength)}{logRecord.LogLevel}");
+                this.WriteLine($"{"LogRecord.TraceFlags:".PadRight(rightPaddingLength)}{logRecord.TraceFlags}");
+                this.WriteLine($"{"LogRecord.State:".PadRight(rightPaddingLength)}{logRecord.State}");
```

Comment:
Isn't this going to potentially output large structured logging objects?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/1613/files#r529127446 Check this and modify `State` printing accordingly.
We can update doc about what does `State` really contain, and how to parse it out.

Reply:
@utpilla Let me know if you need help.

Reply:
The ConsoleExporter is there to provide an easy way for the consumers of the OpenTelemetry Logging to get a sense of what the LogRecord object would contain, right? I think we should definitely output the State to Console because that's the information that the user is trying to log so they would be interested in how it gets logged. For unstructured logging, the state logs the ToString() implementation of the object. For structured logging, the state is populated by the framework and we just output that to console.

If outputting large objects is a concern then, we would have to do fix it for Tags in Activity as well. Activity Tags can also be any object.

We can update the documentation as well.

Reply:
Probably its okay to do ToString() in the console exporter. Lets add a example on what is contained in the state and how to navigate it to a doc.

@ejsmith This okay?

Reply:
Yeah, I guess a large object would get `ToString` called on it and that would just output the type name.